### ISSUE TITLE
GtkWidget deprecation warnings (GTK 3.22)

### ISSUE THREAD

**Issue by daenney** [gotk3/gotk3#290]
I built gotk3/gotk3@628094034cb2 today with GTK 3.22.30 and got greeted by two deprecation warnings for functions on `GtkWidget`

* `override_background_color`:
  ```
  cgo-gcc-prolog: In function ‘_cgo_ee0b896a9dbe_Cfunc_gtk_widget_override_background_color’:
  cgo-gcc-prolog:317:2: warning: ‘gtk_widget_override_background_color’ is deprecated [-Wdeprecated-  declarations]
  In file included from /usr/include/gtk-3.0/gtk/gtkapplication.h:27,
                 from /usr/include/gtk-3.0/gtk/gtkwindow.h:33,
                 from /usr/include/gtk-3.0/gtk/gtkdialog.h:32,
                 from /usr/include/gtk-3.0/gtk/gtkaboutdialog.h:30,
                 from /usr/include/gtk-3.0/gtk/gtk.h:31,
                 from ../../../go/pkg/mod/github.com/gotk3/gotk3@v0.0.0-20180825185631-628094034cb2/gtk/gtk_since_3_20.go:8:
  /usr/include/gtk-3.0/gtk/gtkwidget.h:1148:14: note: declared here
   void         gtk_widget_override_background_color (GtkWidget     *widget,
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  ```
* `override_font`:
  ```
  cgo-gcc-prolog: In function ‘_cgo_ee0b896a9dbe_Cfunc_gtk_widget_override_font’:
  cgo-gcc-prolog:638:2: warning: ‘gtk_widget_override_font’ is deprecated [-Wdeprecated-declarations]
  In file included from /usr/include/gtk-3.0/gtk/gtkapplication.h:27,
                 from /usr/include/gtk-3.0/gtk/gtkwindow.h:33,
                 from /usr/include/gtk-3.0/gtk/gtkdialog.h:32,
                 from /usr/include/gtk-3.0/gtk/gtkaboutdialog.h:30,
                 from /usr/include/gtk-3.0/gtk/gtk.h:31,
                 from ../../../go/pkg/mod/github.com/gotk3/gotk3@v0.0.0-20180825185631-628094034cb2/gtk/label.go:6:
  /usr/include/gtk-3.0/gtk/gtkwidget.h:1153:14: note: declared here
   void         gtk_widget_override_font             (GtkWidget                  *widget,
                ^~~~~~~~~~~~~~~~~~~~~~~~
  ```

**Comment by daenney** [comment]
These are just warnings, they also surfaced in #283 but since the issue there where actual compilation errors I figured it would make sense to have a separate issue for these.

**Comment by stdevYuniers** [comment]
I got same error when try install it as dependency of `skycoin cx`

@Bios-Marcel please see:

- [Error](https://ci.appveyor.com/project/stdevYuniers/cx/builds/20971791#L543)
- [appveyor config file](https://github.com/skycoin/cx/blob/master/.appveyor.yml)

**Comment by Bios-Marcel** [comment]
Maybe there is aproblem withmy guide, I'll spin up avm tmr in order to test it

**Comment by stdevYuniers** [comment]
> In the meantime you can check this: https://github.com/UwUNote/uwunote/blob/master/appveyor.yml
> 
> That's deffo a working example.

Thank you, I will test again.

The only mayor difference between both `appveyor files` are use of `Visual Studio 2015` (you) and `Visual Studio 2017` (me).

Maybe use of mayor version of gcc can cause build fail.

**Comment by stdevYuniers** [comment]
> Okay ... so I reran the last build of `UwUNote`. Guess what ... it fails: https://ci.appveyor.com/project/Bios-Marcel/uwunote
> 
> This is like the third time the windows build broke over night, without me making any changes.
> 
> I assume I might need some more time to see what happens here.

I'll wait, I understand, Windows compilation always cause headaches.

**Comment by Bios-Marcel** [comment]
@stdevYuniers You cancreate anew issue,in case your problem still persist, I amgoing to close this one, since its solved.

**Comment by Bios-Marcel** [comment]
Solved with #316
